openapi.json

```diff
@@ -2009,60 +2113,6 @@
               "type": "integer"
             },
             "example": 42
-          },
-          {
-            "name": "start_date",
```

Comment:
Why did you remove these query params?

Reply:
From discussion in Slack: these were mistakenly documented for the create action so are just being cleaned up here.

Reply:
The query params were incorrectly added to the create action.